runtime-config-linux.md

```diff
@@ -201,6 +201,8 @@ The following parameters can be specified to setup the controller:
 
 * **`kernel`** *(uint64, optional)* - sets hard limit for kernel memory
 
+* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
```

Comment:
On Mon, Apr 11, 2016 at 03:54:09PM -0700, Vish Kannan wrote:
> > +* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
> 
> Oops.. @wking Can you post a PR to fix this ?

@vbatts just pushed c1612af → d4e7326 in #370 which should cover this.